src/Orc.NuGetExplorer.Xaml/ViewModels/PackageDetailsViewModel.cs

```diff
@@ -155,10 +151,17 @@ private async Task OnInstallPackageExecuteAsync()
 
         private bool OnInstallPackageCanExecute()
         {
-            var anyProject = NuGetActionTarget?.IsValid ?? false;
+            if (!IsPackageApplied)
```

Comment:
Shouldn't we check `if (IsPackageApplied) { return false; }` instead?

Reply:
If that's not the case, then what does "IsPackageApplied" represent?

Reply:
It's represents is metadata information applied to current model for newly selected package.
If not we should not allow to do any actions until loading is finish (don't have all requirements)